pkg/util/logs/logs.go

```diff
@@ -44,12 +44,22 @@ func (writer GlogWriter) Write(data []byte) (n int, err error) {
 
 // InitLogs initializes logs the way we want for kubernetes.
 func InitLogs() {
+	logFlushFreq := pflag.Duration("log-flush-frequency", DefaultLogFlushFreq, "Maximum number of seconds between log flushes")
```

Comment:
@errordeveloper Since you've moved this out into a separate PR, let's remove it from here